docs/docs/docs/01-core/strapi/commands/plugin/05-link-watch.md

```diff
@@ -0,0 +1,33 @@
+---
+title: plugin:watch:link
+description: An in depth look at the plugin:watch:link command of the Strapi CLI
+tags:
+  - CLI
+  - commands
+  - plugins
+---
+
+The `plugin:watch:link` command recompiles your plugin and pushes those changes to your local yalc registry to simulate using your plugin as a node_module in another project.
```

Comment:
Probably an annoying comment.
Is the order of the command normally relevant? I would see a possibility to only "link" a plugin and not "watch" for changes, and because of that I would see `plugin:link:watch` more fitting than `plugin:watch:link` . 

just writting it here to get it out of my head 🤷‍♂️ , but feel free to ignore this 😄

Reply:
i did originally have it as `link-watch` but @innerdvations asked to change it because it runs watch & links at the same time. Which is the same idea you're having but inverted.

Reply:
Yeah, I debated that myself, I could probably be convinced either way as long as it's consistent with other commands

Reply:
I dont mind either way! I will leave that responsibility to josh 😛

Reply:
I'm gonna leave it as is, because i agreed with ben to begin with lol